lakefile.lean

```diff
@@ -66,6 +66,15 @@ lean_exe shake where
   root := `Shake.Main
   supportInterpreter := true
 
+/--
+`lake exe pole` queries the Mathlib speedcenter for build times for the current commit,
+and then calculates the longest pole
+(i.e. the sequence of files you would be waiting for during a infinite parallelism build).
+-/
+lean_exe pole where
+  root := `LongestPole.Main
```

Comment:
Sorry, I keep on changing my mind as I discover more about Lean executables: Could you rename the file to `scripts/longestPole.lean` and change the lakefile to use `srcDir` rather than `root`?```suggestion
  srcDir := "scripts"
```

Reply:
Again, I am in favor of all the executables being located in the same place and being declared in the same way. This sounds fine to me, but I think it would be best to do this for all the executables simultaneously in one PR to make sure they are all in harmony with each other.

Reply:
That's already the case, see `checkYaml`

Reply:
Right, and `Shake` is currently the way `LongestPole` is now. I stand by saying that these organizational issues should be dealt with in a separate PR, where they can be reviewed separately.

Reply:
I split this into multiple files, so now I think this is moot